app-admin/vaultwarden/files/vaultwarden-envfile-1.33.2.patch

```diff
@@ -0,0 +1,63 @@
+As upstream defaults create directories where binary is executed (/usr/bin/), this patch is needed to change paths to Gentoo specific locations
+--- a/.env.template
++++ b/.env.template
+@@ -5,33 +5,29 @@
+ ## Be aware that most of these settings will be overridden if they were changed
+ ## in the admin interface. Those overrides are stored within DATA_FOLDER/config.json .
+ ##
+-## By default, Vaultwarden expects for this file to be named ".env" and located
+-## in the current working directory. If this is not the case, the environment
+-## variable ENV_FILE can be set to the location of this file prior to starting
+-## Vaultwarden.
+ 
+ ####################
+ ### Data folders ###
+ ####################
+ 
+ ## Main data folder
+-# DATA_FOLDER=data
++DATA_FOLDER=/var/lib/vaultwarden
+ 
+ ## Individual folders, these override %DATA_FOLDER%
+-# RSA_KEY_FILENAME=data/rsa_key
+-# ICON_CACHE_FOLDER=data/icon_cache
+-# ATTACHMENTS_FOLDER=data/attachments
+-# SENDS_FOLDER=data/sends
+-# TMP_FOLDER=data/tmp
++# RSA_KEY_FILENAME=/var/lib/vaultwarden/rsa_key
```

Comment:
if this were shell-sourced file, we could simply write these as `"${DATA_FOLDER}"/rsa_key` etc, which doesn't reduce the patch size, but it does reduce the repetition somewhat.